src/views/Signup/components/SignupBlog.tsx

```diff
@@ -0,0 +1,56 @@
+import { SignupComponent } from './AuthView';
+import InternxtDevices from '../../../assets/images/banner/internxt_secure_cloud_storage.webp';
+import { Helmet } from 'react-helmet-async';
+import envService from 'app/core/services/env.service';
+import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+
+const SignupAuth = () => {
+  return (
+    <div className="flex h-52 w-full flex-col space-y-2 lg:items-start lg:pt-0">
+      <SignupComponent
+        buttonColor="bg-primary focus-visible:bg-primary-dark active:bg-primary-dark"
+        appRedirect={true}
+        variant="blog"
+      />
+    </div>
+  );
+};
+
+export default function SignupBlog(): JSX.Element {
```

Comment:
If several components are related, for example sign up blog and derivates, create inside componentes, a SignUpBlog directory in order to keep it together. In that case will be easier to know the relation between them